apps/communications/contacts/locales/contacts.en-US.properties

```diff
@@ -265,3 +265,12 @@ edit-picture = Edit Picture
 remove-photo = Remove photo
 change-photo = Change photo
 photo-options = Photo options
+
+# Importing from multiple vCard
+vCardContacts-imported         = {[ plural(n) ]}
+vCardContacts-imported[zero]   = No new contacts imported
+vCardContacts-imported[one]    = Imported one contact
+vCardContacts-imported[two]    = Imported {{n}} contacts
+vCardContacts-imported[few]    = Imported {{n}} contacts
+vCardContacts-imported[many]   = Imported {{n}} contacts
+vCardContacts-imported[other]  = Imported {{n}} contacts
```

Comment:
I searched but I just found specific strings that referenced memory card or sim, for example. I didn't find a generic enough string.